app/views/targets/index.json.jbuilder

```diff
@@ -0,0 +1,3 @@
+json.targets @targets do |target|
```

Comment:
I think it's better to use `json.targets @targets, partial: 'info', as: :target`

Reply:
:+1: